ui/lib/data-fetch.js

```diff
@@ -7,21 +7,25 @@ const dataFetch = (url, options = {}, successFn, errorFn) => {
   // setTimeout(() => controller.abort(), 10000); // nice to have but will mess with the load test
   fetch(url, options)
     .then(res => {
-      if (res.status === 401 || res.redirected){
-        if (window.location.host.endsWith('3000')){
+      if (res.status === 401 || res.redirected) {
+        if (window.location.host.endsWith('3000')) {
```

Comment:
I recognize this isn't part of your changes, but can you confirm this logic is dealing with whether or not Meshery UI is running in development mode? If so, let's open an issue to track this mode globally. // @dhruv0000 @sudo-NithishKarthik

Reply:
Just checked it, UI dev server is loading gracefully. Refreshed UI lot of times on chrome browser seems to be loading efficiently. But still not sure that the issue with dev server is fully fixed.